2018-12-17 18:19:36,307 ERROR: Exception on /auth/register [GET] [in c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Marlon Pimentel\microblog\app\auth\routes.py", line 50, in register
    form=form)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Marlon Pimentel\microblog\app\templates\auth\register.html", line 6, in template
    <h1>{{ _('Register') }}</h1>
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 300, in parse_import
    self.stream.expect('name:as')
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\lexer.py", line 384, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token 'as', got 'Register'
2018-12-17 18:31:58,318 INFO: Microblog startup [in C:\Users\Marlon Pimentel\microblog\app\__init__.py:75]
2018-12-17 18:32:08,138 ERROR: Exception on /auth/reset_password_request [GET] [in c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Marlon Pimentel\microblog\app\auth\routes.py", line 65, in reset_password_request
    title=_('Reset Password'), form=form)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\templating.py", line 57, in get_source
    return self._get_source_fast(environment, template)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\templating.py", line 85, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: auth/reset_password_request.html
2018-12-17 18:34:34,479 INFO: Microblog startup [in C:\Users\Marlon Pimentel\microblog\app\__init__.py:75]
2018-12-25 23:41:28,876 INFO: Microblog startup [in C:\Users\Marlon Pimentel\microblog\app\__init__.py:75]
2018-12-26 00:38:09,752 INFO: Microblog startup [in C:\Users\Marlon Pimentel\microblog\app\__init__.py:75]
2018-12-26 19:38:24,651 INFO: Microblog startup [in C:\Users\Marlon Pimentel\microblog\app\__init__.py:75]
2018-12-30 13:38:50,512 INFO: Microblog startup [in C:\Users\Marlon Pimentel\microblog\app\__init__.py:75]
